### ISSUE TITLE
Add reduced-motion support

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#1012]
**Is your feature request related to a problem? Please describe.**

Windows and MacOS have an accessibility setting which should set the `prefers-reduced-motion` media query. Any animations or transitions should be updated to respect this preference.

https://www.a11ywithlindsey.com/blog/reducing-motion-improve-accessibility/

**Describe the solution you'd like**

All animations and transitions to be disabled when this media query is present.

Including (but not limited to):

* Sign in video background
* Button hover transitions
* Primary navigation, sidebar help and table detail flouts
* Loading spinners
